vscode/src/services/AuthProvider.ts

```diff
@@ -206,9 +206,14 @@ export class AuthProvider {
             ...options
         )
         switch (option) {
-            case 'Manage Account':
-                void vscode.env.openExternal(vscode.Uri.parse(ACCOUNT_USAGE_URL.toString()))
+            case 'Manage Account': {
+                // Add the username to the web can warn if the logged in session on web is different from VS Code
+                const uri = vscode.Uri.parse(ACCOUNT_USAGE_URL.toString()).with({
+                    query: `username=${this.authStatus.username}`,
```

Comment:
```suggestion
                    query: `username=${encodeURIComponent(this.authStatus.username)}`,
```

This _might_ need URI encoding (I’m not sure if `foo&bar` would be a valid username 🙈 ) otherwise

Reply:
I think its fine because `A username consists of letters, numbers, hyphens (-), dots (.), underscore (_) and may not begin or end with a dot, nor begin with a hyphen.`

Reply:
I added `encodeURIComponent` just in case the username rules change or we sub it out for some other `authStatus` property